snc-library.sh

```diff
@@ -143,11 +143,12 @@ function create_json_description {
             | ${JQ} ".clusterInfo.openshiftVersion = \"${OPENSHIFT_RELEASE_VERSION}\"" \
             | ${JQ} ".clusterInfo.clusterName = \"${SNC_PRODUCT_NAME}\"" \
             | ${JQ} ".clusterInfo.baseDomain = \"${BASE_DOMAIN}\"" \
-            | ${JQ} ".clusterInfo.appsDomain = \"apps-${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\"" >${INSTALL_DIR}/crc-bundle-info.json
+            | ${JQ} ".clusterInfo.appsDomain = \"apps.${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\"" >${INSTALL_DIR}/crc-bundle-info.json
     if [ ${bundle_type} == "snc" ] || [ ${bundle_type} == "okd" ]; then
         openshiftInstallerVersion=$(${OPENSHIFT_INSTALL} version)
         tmp=$(mktemp)
-        cat ${INSTALL_DIR}/crc-bundle-info.json | ${JQ} ".buildInfo.openshiftInstallerVersion = \"${openshiftInstallerVersion}\"" \
+        cat ${INSTALL_DIR}/crc-bundle-info.json | \
+            ${JQ} ".buildInfo.openshiftInstallerVersion = \"${openshiftInstallerVersion}\" | .clusterInfo.appsDomain |= \"apps-${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\"" \
```

Comment:
Don't you need a `${JQ}` after the `|`? And can you split this line in 2?

Reply:
> Don't you need a ${JQ} after the |?

No 

> And can you split this line in 2?

`".buildInfo.openshiftInstallerVersion = \"${openshiftInstallerVersion}\" | .clusterInfo.appsDomain |= \"apps-${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\""` is a single JQ query so can't split in 2 different line.

The CI is failed not during bundle creation but because of https://github.com/crc-org/crc/pull/3806 one

Reply:
The block above is using `JQ` after every pipe, I'd prefer we keep this consistent.

Reply:
@cfergeau updated.